apps/chat/src/constants/validation-helpers.ts

```diff
@@ -0,0 +1,74 @@
+import { isVersionPartSizeValid, isVersionValid } from '@/src/utils/app/common';
+import { doesHaveNotAllowedSymbols } from '@/src/utils/app/file';
+
+import { MIME_FORMAT_REGEX } from '@/src/constants/file';
+import {
+  formErrors,
+  urlErrors,
+  versionsErrors,
+} from '@/src/constants/form-errors';
+
+import { z as zodValidation } from 'zod';
+
+export const MarketplaceEntityBaseSchema = zodValidation.object({
+  name: zodValidation
+    .string()
```

Comment:
I suggest to add `trim()` to prohibit "double" spaces names
```suggestion
    .string()
    .trim()
```